torch/_inductor/codegen/cpp_gemm_template.py

```diff
@@ -121,6 +107,77 @@
         Mt_blocks * Nt_blocks * Kt_blocks * {{num_threads}} >= Mr_blocks * Nr_blocks * Kr_blocks,
         "Not all partitions are assigned."
     );
+"""
+
+GEMM_TEMPLATE_MULTI_THREADS_PARAM = r"""
```

Comment:
using plural is better? same for other var/func names.
```suggestion
GEMM_TEMPLATE_MULTI_THREADS_PARAMS = r"""
```

Reply:
Thanks for the comment and change to plural format.